tests/test_server.py

```diff
@@ -1323,3 +1323,10 @@ async def test_resource_deploy_done_error_handling(server, client, environment,
         change=const.Change.nochange,
     )
     assert result.code == 404, result.result
+
+
+@pytest.mark.asyncio
+async def test_start_location_no_redirect(server):
```

Comment:
would expect a test that checks what a request to "/" produces, this testcase is not very clear

Reply:
How could this be done?

Reply:
1. the server fixture sets up the server, so that is OK
2. you need to know where the server listens. If you look into conftest.py, you see:

```
    config.Config.set("server", "bind-port", port)
    config.Config.set("server", "bind-address", "127.0.0.1")
```

3. to get the port you can do .e.g. 

```
inmanta.server.config.server_bind_port.get()
```

4. then you can use an (async) httpclient to connect to the server and make the request (e.g. like the testcase test_resources_paging, which basically does all of this)

Reply:
@wouterdb I rewrote the test. it is indeed a lot clearer this way. Thanks!